cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -658,7 +658,9 @@ Resources:
     Properties:
         ActionsEnabled: true
         AlarmActions:
-            - !If [SilenceAlertsCondition, !Ref AWS::NoValue, !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:javabuilder-high-usage"]
+            # TODO: after we have run at high usage for a while, consider re-enabling this alarm. Right now it is too noisy
```

Comment:
More a comment for when we turn this back on, but is the problem that our usage is increasing and that is what is making the alarm too noisy, or that the anomaly detection band isn't wide enough? Do you have a Jira ticket for this where I could add this comment?

Reply:
Maybe it'll be less spiky with more ppl using it, but feel like spikiness will probably persist, especially in lower usage times (summer/holidays/weekends/nights/etc)

Reply:
Right now I think it's some of both. This alarm may not be useful as-is, and we may just want an alarm if we are getting close to our reserved concurrency instead. I'll make a jira ticket

Reply:
https://codedotorg.atlassian.net/browse/JAVA-648